content/participants/ttjc@jittash/index.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Jittash
+type: "participant"
+name: "Jittash Raz Mandhyannee"
+portfolio: 
+github-link: "https://github.com/jittash"
+linkedin-link: "https://www.linkedin.com/in/jittash-r-mandhyannee-21b93915b"
+looking-for: "internship"
+description: "PHP| JS | Node | Learning React"
+---
+
+Hi! I am Jittash Raz Mandhyannee,pre final year Btech student at Bhopal let me walk you through my intro, projects and blogs.
+
+### introduction
+
+I am a self motivated web developer who constantly improves and tries to upgrade his skills.
+
+## why are you taking this challenge?
+
+I will learn and experience more by creating projects which will be helpful for me to land an internship this summer(2020).
+
+## something interesting about you?
+
+I have a twin brother.
+I have done various social wefare and other events since i joined college and connected with people of various fields. 
+
+### help I need?
+
+Being guided by mentors so as to use my skills in the best possible way.
+
+### how to contact me?
+
+- LinkedIn - "https://www.linkedin.com/in/jittash-r-mandhyannee-21b93915b
+
+### projects
```

Comment:
You have mentioned some of the projects provide the hosted link and source code for each one, also make sure projects are made after mid-March 2020